components/various.py

```diff
@@ -60,7 +61,8 @@ def compute_various(df, options):
 
         pod = {"date": date}
 
-        get_extension = lambda x: "webp" if x in [48, 49, 50, 51, 52, 53, 60, 61] else "png"
+        # get_extension = lambda x: "webp" if x in [48, 49, 50, 51, 52, 53, 60, 61] else "png"
+        get_extension = "png"
```

Comment:
`get_extension` is a function. As you see in the following line it's called with the "spot" argument. You have to modify that as well, otherwise it won't work.

Sounds pretty nice that we have all images in png, I was quite annoyed with what I had to add here.